client/components/accountoverview/accountOverview.tsx

```diff
@@ -62,6 +69,18 @@ const AccountOverviewRenderer = ([mdaResponse, cancelledProductsResponse]: [
 		(_) => _.alertText,
 	);
 
+	const hasDigiSubAndContribution =
+		allActiveProductDetails.some((productDetail) =>
+			isSpecificProductType(productDetail, PRODUCT_TYPES.contributions),
```

Comment:
Could you use an OR here rather than looping through all of the active products twice? eg.

`isSpecificProductType(productDetail, PRODUCT_TYPES.contributions) || isSpecificProductType(productDetail, PRODUCT_TYPES.digipack)`

Reply:
I thought about this, but wouldn't that be true if you had one or the other but not necessarily both?

Reply:
Ah, sorry I misread the existing logic 🤦  Yes, you're absolutely right as you need to check that they have _both_ products 👍